openprescribing/pipeline/management/commands/bigquery_upload.py

```diff
@@ -1,14 +1,29 @@
 from __future__ import print_function
 
 from django.core.management import BaseCommand
+from django.db.models import Max
 
 from ebmdatalab import bigquery
+from frontend.models import PracticeStatistics, Prescription
 
 from ...cloud_utils import CloudHandler
 
 
 class Command(BaseCommand):
     def handle(self, *args, **kwargs):
+        # Make sure that PracticeStatistics and Prescription tables both have
+        # latest data.
+        latest_practice_statistic_date = PracticeStatistics.objects\
+            .aggregate(Max('date'))['date__max']
+        latest_prescrption_date = Prescription.objects\
```

Comment:
can we rename the variable `latest_prescrption_date` -> `latest_prescription_date`